modules/orientdb/src/test/java/org/testcontainers/containers/OrientDBContainerTest.java

```diff
@@ -0,0 +1,86 @@
+package org.testcontainers.containers;
+
+import com.orientechnologies.orient.core.db.ODatabaseSession;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+/**
+ * @author robfrank
+ */
+public class OrientDBContainerTest {
+
+    @Test
+    public void testContainerLifecycle() {
+        OrientDBContainer container = new OrientDBContainer();
+
+        container.start();
+
+        assertThat(container.isRunning()).isTrue();
+
+        container.stop();
+
+        assertThat(container.isRunning()).isFalse();
+    }
```

Comment:
I think this test is unnecessary, since it's a feature of Testcontainers core. Can we remove?

Reply:
removed